src/main/resources/openapiv2.json

```diff
@@ -671,75 +802,130 @@
+                    "type": "string"
                 },
-                "offset": {
-                    "type": "integer"
+                "auto.offset.reset": {
+                    "description": "Resets the offset position for the consumer. \nIf set to earliest, messages are read from the first offset. \nIf set to latest, messages are read from the latest offset.",
+                    "type": "string"
                 },
-                "topic": {
+                "auto.commit.enable": {
+                    "description": "If set to true, message offsets are committed automatically for the consumer. If set to false, message offsets must be committed manually.",
                     "type": "string"
                 },
-                "value": {
-                    "type": "object"
+                "fetch.min.bytes": {
+                    "description": "Sets the minimum ammount of data in bytes for the consumer to receive. The broker waits until the data to send exceeds this amount.",
+                    "type": "string"
                 },
-                "key": {
-                    "type": "object"
+                "consumer.request.timeout.ms": {
+                    "description": "Sets the maximum amount of time in milliseconds (ms) for the consumer to wait for messages for a request. If the timeout period is reached without a response, an error is returned.",
+                    "type": "string"
                 }
+            },
+            "example": {
+                "name": "consumer1",
+                "format": "binary",
+                "auto.offset.reset": "earliest",
+                "auto.commit.enable": "false",
+                "fetch.min.bytes": "512",
+                "consumer.request.timeout.ms": "30000"
             }
         },
-        "Topics": {
-            "title": "Topics",
+        "Offsets": {
+            "title": "Offsets",
             "type": "object",
             "properties": {
-                "topics": {
+                "offsets": {
                     "type": "array",
                     "items": {
-                        "type": "string"
+                        "$ref": "#/definitions/Offset"
                     }
                 }
             },
-            "example": "{\n    \"topics\": [\n        \"topic1\",\n        \"topic2\"\n    ]\n}"
+            "example": {
+                "offsets": [
+                    {
+                        "topic": "topic",
+                        "partition": 0,
+                        "offset": 15
+                    },
+                    {
+                        "topic": "topic",
+                        "partition": 1,
+                        "offset": 42
+                    }
+                ]
+            }
         },
-        "Records": {
-            "title": "Records",
+        "OffsetsWithoutTopic": {
```

Comment:
It's used in the HTTP response related to records sent, so I would call it OffsetsRecordsSent